### PR TITLE
fix(tabs): add styles for -justified tab classes

### LINKED ISSUE #5243: Missing bottom border on uib-tabset inactive tabs with justified=true
When the uib-tabset is configured with justified=true, the bottom border of the inactives tabs is not visible. This is reproductible on the demo page.

For justified tabs, Bootstrap display the bottom border on each individual tabs but this CSS is missing as div are used for tabs instead of a.

### PR DESCRIPTION
**Pull request by wesleycho**
- Add styles for when the user is using `.nav-justified` and
  `.nav-tabs-justified`

This fixes #5243.

### CODE REVIEW

File: src/tabs/tabs.css [hunk 1]
```diff
@@ -44,4 +44,33 @@
...
+  }
+}
+.nav-tabs.nav-justified > li > div {
+  margin-right: 0;
+  border-radius: 4px;
+}
+.nav-tabs.nav-justified > .active > div,
+.nav-tabs.nav-justified > .active > div:hover,
+.nav-tabs.nav-justified > .active > div:focus {
+  border: 1px solid #ddd;
```
Comment (icfantv) [inline src/tabs/tabs.css]: do we want `cursor: pointer;` here as well?

Comment (icfantv) [inline src/tabs/tabs.css]: nevermind.  i think we're ok...

**Comment by wesleycho** [comment]
This PR may be rejected by discussion in #5254

**Comment by wesleycho** [comment]
Closing as superceded by the culmination of #5262.

**Pull request closed without merging**